homeassistant/components/lg_netcast/media_player.py

```diff
@@ -79,19 +107,39 @@ class LgTVDevice(MediaPlayerEntity):
     _attr_assumed_state = True
     _attr_device_class = MediaPlayerDeviceClass.TV
     _attr_media_content_type = MediaType.CHANNEL
+    _attr_has_entity_name = True
+    _attr_name = None
 
-    def __init__(self, client, name, on_action_script):
+    def __init__(self, client, name, model, *, unique_id=None):
```

Comment:
There's no reason the unique_id should be optional?
```suggestion
    def __init__(self, client, name, model, unique_id):
```

Reply:
Agreed - this is left from the original implementation that didn't migrate the YAML (and discover the unique_id)

Reply:
Done